kitsune/sumo/views.py

```diff
@@ -120,183 +112,6 @@ def test_memcached(host, port):
-        (settings.IMAGE_UPLOAD_PATH, os.R_OK | os.W_OK, msg),
-        (settings.THUMBNAIL_UPLOAD_PATH, os.R_OK | os.W_OK, msg),
-        (settings.GALLERY_IMAGE_PATH, os.R_OK | os.W_OK, msg),
-        (settings.GALLERY_IMAGE_THUMBNAIL_PATH, os.R_OK | os.W_OK, msg),
-        (settings.GALLERY_VIDEO_PATH, os.R_OK | os.W_OK, msg),
-        (settings.GALLERY_VIDEO_THUMBNAIL_PATH, os.R_OK | os.W_OK, msg),
-        (settings.GROUP_AVATAR_PATH, os.R_OK | os.W_OK, msg),
-    )
-
-    filepath_results = []
-    for path, perms, notes in filepaths:
-        path = os.path.join(settings.MEDIA_ROOT, path)
-        path_exists = os.path.isdir(path)
-        path_perms = os.access(path, perms)
-
-        if path_exists and path_perms:
-            filepath_results.append(
-                (INFO, "%s: %s %s %s" % (path, path_exists, path_perms, notes))
-            )
-
-    status["filepaths"] = filepath_results
-
-    # Check RabbitMQ.
-    rabbitmq_results = []
-    try:
-        rabbit_conn = establish_connection(connect_timeout=5)
-        rabbit_conn.connect()
-        rabbitmq_results.append((INFO, "Successfully connected to RabbitMQ."))
-    except (socket.error, IOError) as exc:
-        rabbitmq_results.append((ERROR, "Error connecting to RabbitMQ: %s" % str(exc)))
-
-    except Exception as exc:
-        rabbitmq_results.append((ERROR, "Exception while looking at RabbitMQ: %s" % str(exc)))
-
-    status["RabbitMQ"] = rabbitmq_results
-
-    # Check ES.
-    es_results = []
-    try:
-        es_utils.get_doctype_stats(es_utils.all_read_indexes()[0])
-        es_results.append((INFO, ("Successfully connected to ElasticSearch and index " "exists.")))
-
-    except es_utils.ES_EXCEPTIONS as exc:
-        es_results.append((ERROR, "ElasticSearch problem: %s" % str(exc)))
-
-    except Exception as exc:
-        es_results.append((ERROR, "Exception while looking at ElasticSearch: %s" % str(exc)))
-
-    status["ElasticSearch"] = es_results
-
-    # Check Celery.
-    # start = time.time()
-    # pong = celery.task.ping()
-    # rabbit_results = r = {'duration': time.time() - start}
-    # status_summary['rabbit'] = pong == 'pong' and r['duration'] < 1
-
-    # Check Redis.
-    redis_results = []
-    if hasattr(settings, "REDIS_BACKENDS"):
-        for backend in settings.REDIS_BACKENDS:
-            try:
-                redis_client(backend)
-                redis_results.append((INFO, "%s: Pass!" % backend))
-            except RedisError:
-                redis_results.append((ERROR, "%s: Fail!" % backend))
-    status["Redis"] = redis_results
-
-    status_code = 200
-
-    status_summary = {}
-    for component, output in list(status.items()):
-        if ERROR in [item[0] for item in output]:
-            status_code = 500
-            status_summary[component] = False
-        else:
-            status_summary[component] = True
-
-    return render(
-        request,
-        "services/monitor.html",
```

Comment:
`kitsune/sumo/jinja2/services/monitor.html` can be removed, and I think `kitsune/sumo/static/sumo/css/monitor.css` can also be removed